docs/cli_installation.md

```diff
@@ -94,14 +74,14 @@ VERSION=$(curl --silent "https://api.github.com/repos/argoproj/argo-cd/releases/
 Replace `VERSION` in the command below with the version of Argo CD you would like to download:
 
 ```bash
-curl -sSL -o argocd-darwin-amd64 https://github.com/argoproj/argo-cd/releases/download/$VERSION/argocd-darwin-amd64
+curl -sSL -o argocd https://github.com/argoproj/argo-cd/releases/download/$VERSION/argocd-darwin-$(uname -m)
```

Comment:
```suggestion
curl -sSL -o argocd https://github.com/argoproj/argo-cd/releases/download/$VERSION/argocd-darwin-$(uname -m)
```

This is wrong. `uname -m` would not give the amd64 or arm64 response due to which the link becomes invalid.

Reply:
sorry about that. since we are modifying it to use `Apple Silicon` Mac users, I am going to change it to `arm64` which is by default for users.